configure.ac

```diff
@@ -136,7 +136,7 @@ dnl AC_FUNC_REALLOC
 AC_TYPE_SIGNAL
 AC_FUNC_STAT
 AC_FUNC_STRFTIME
-AC_CHECK_FUNCS([issetugid inet_pton])
+AC_CHECK_FUNCS([issetugid])
```

Comment:
No problem. Thanks for your reply.
Regarding C99, Haiku/x86_gcc2 still uses it because the core developers chose to maintain binary compatibility with BeOS. But that will be dropped someday.
Anyway, thank you very much for the **`AC_SEARCH_LIBS(accept,network)`**. This means lighttpd 1.4.41 won't need any patch on Haiku/x86{,_64} which are already using gcc 5.3 ;-)